### PR TITLE
correct description of `skipMiddlewareUrlNormalize` in advanced middleware flags

### PR DESCRIPTION
**Pull request by Evavic44**
I think there's some form of typographical error in the `skipMiddlewareUrlNormalize` line under the advanced middleware flags section of the docs.

This is the line from the docs [Source link](https://nextjs.org/docs/app/building-your-application/routing/middleware#advanced-middleware-flags):

> `skipMiddlewareUrlNormalize` allows disabling the URL normalizing Next.js does to make handling direct visits and client-transitions the same. There are some advanced cases where you need full control using the original URL which this unlocks.

Corrected to:

> `skipMiddlewareUrlNormalize` allows disabling URL normalizing in Next.js to make handling direct visits and client-transitions the same. There are some advanced cases where you need full control using the original URL which this unlocks.

### CODE REVIEW

File: docs/02-app/01-building-your-application/01-routing/13-middleware.mdx [hunk 1]
```diff
@@ -413,7 +413,7 @@ export default async function middleware(req) {
 }
 ```
 
-`skipMiddlewareUrlNormalize` allows disabling the URL normalizing Next.js does to make handling direct visits and client-transitions the same. There are some advanced cases where you need full control using the original URL which this unlocks.
+`skipMiddlewareUrlNormalize` allows disabling URL normalizing in Next.js to make handling direct visits and client-transitions the same. There are some advanced cases where you need full control using the original URL which this unlocks.
```
Comment (samcx) [inline docs/02-app/01-building-your-application/01-routing/13-middleware.mdx:416]:
```suggestion
`skipMiddlewareUrlNormalize` allows for disabling the URL normalization in Next.js to make handling direct visits and client-transitions the same. In some advanced cases, this option provides full control by using the original URL.
```

**Review by samcx — CHANGES_REQUESTED** [review]

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 41e470ae37a2a80573cd11aed57f8464bf5488d3

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 41e470ae37a2a80573cd11aed57f8464bf5488d3

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 41e470ae37a2a80573cd11aed57f8464bf5488d3

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by samcx — APPROVED** [review]
<picture data-single-emoji=":lgtm:" title=":lgtm:">:lgtm:</picture>

**Pull request merged**